gallery/workbook/Azure Monitor SCOM managed instance.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
is this relative to a resource type?  are you passing a scom mi resource id as ComponentId when openign the view?  if so, then this gallery file should just be the resource type instead, like the other files in the folder.

yours would probably be `microsoft.scom-managedinstances.json`

https://github.com/microsoft/Application-Insights-Workbooks/blob/master/Documentation/Contributing.md#how-to-create-and-name-a-gallery-file

The only reason things would usually have a name that includes azure monitor (or anything not a resource type) are for workbook galleries on things that are *not* resources.  (Like "Azure monitor" is not "a" resource)

Reply:
Done changed the name to suggested one "microsoft.scom-managedinstances", removed Azure monitor from name
Also pasted image of the testing and workbook is showing up in the scommi resource itself in workbook TOC.